app/packages/looker-3d/src/fo3d/mesh/Ply.tsx

```diff
@@ -21,6 +23,45 @@ interface PlyProps {
   children?: React.ReactNode;
 }
 
+const PlyWithPointsMaterial = ({
+  name,
+  geometry,
+  defaultMaterial,
+}: {
+  name: string;
+  geometry: BufferGeometry;
+  defaultMaterial: FoMeshMaterial;
+}) => {
+  const overrideMaterial = {
+    shadingMode: "height",
+    customColor: defaultMaterial["color"] ?? "#ffffff",
+    pointSize: 2,
+    attenuateByDistance: false,
+    opacity: defaultMaterial.opacity,
+  } as FoPointcloudMaterialProps;
+
+  const pointsContainerRef = useRef();
+
+  const pointsMaterialJsx = usePcdMaterial(
+    name,
+    geometry,
+    overrideMaterial,
+    pointsContainerRef
+  );
+
+  const mesh = useMemo(() => new Points(geometry), [geometry]);
+
+  if (!geometry || !pointsMaterialJsx) {
```

Comment:
Nit: `pointsMaterialJsx` => `pointsMaterialElement` since a `<Foo />` is a `React.createElement(Foo)`